var/spack/repos/builtin/packages/npb/package.py

```diff
@@ -22,9 +22,19 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
+import numbers
+
 from spack import *
 
 
+def is_integral(x):
```

Comment:
@hartzell I was thinking about that. If I am moving the function to `variant.py` I would like to agree on naming. Is something like `is_integral` good enough (focus on the predicate) or do you prefer something like `any_integer` (more declarative)? @adamjstewart @davydden @citibeth